client-cli/src/command/transaction_command.rs

```diff
@@ -1009,13 +1024,97 @@ fn ask_transfer_address() -> Result<ExtendedAddr> {
     Ok(address)
 }
 
-fn ask_node_metadata() -> Result<CouncilNode> {
+fn keypackage_info(keypackage: &KeyPackage) -> Result<String> {
+    let mut credential: Vec<u8> = vec![];
+    keypackage.payload.credential.encode(&mut credential);
+    let extensions = keypackage
+        .payload
+        .extensions
+        .iter()
+        .map(|e| {
+            let mut extension_entry: Vec<u8> = vec![];
+            e.encode(&mut extension_entry);
+            base64::encode(&extension_entry)
+        })
+        .collect::<Vec<_>>();
+    let credential = base64::encode(&credential);
+    let l = credential.len();
+    let credential_info = if l > 64 {
+        format!("{}........{}", &credential[0..32], &credential[l - 32..])
+    } else {
+        credential
+    };
```

Comment:
instead of displaying credential as base64, it may be better to parse the X509 in it + extract the quote and show some information from it -- you can use https://github.com/crypto-com/chain/blob/master/chain-tx-enclave-next/enclave-ra/ra-client/src/verifier.rs#L127 for it.
I guess here, for display, it could either display the extracted details, or the potential detected error